benchmarks/README.md

```diff
@@ -0,0 +1,22 @@
+# cuGraph benchmarks
+
+## Overview
+
+This directory contains source and configuration files for benchmarking
+`cuGraph`.  The sources are currently intended to benchmark `cuGraph` via the
+python API, but this is not a requirement, and future updates may include
+benchmarks written in C++ or other languages.
+
+The benchmarks here assume specific datasets are present in the `datasets` directory under the root of the `cuGraph` source tree.
+
+## Prerequisites
+
+* `pytest` and the `pytest-benchmark` plugin
+* cugraph built and installed (or `cugraph` sources and built C++ extensions available on `PYTHONPATH`)
+* *< show dependencies needed for GPU metrics (should be just pynvml) >*
+* *< show optional dependencies to install for plotting >*
+
+## Usage
+
```

Comment:
We should document how to quickly run:
- a specific set of graphs
- a specific set of algo

These reqs will keep coming.

Reply:
good idea. I need to update the README for this anyway so I'll resolve this in the next commit to this PR.